Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusPublicInterface.h

```diff
@@ -56,6 +66,14 @@ namespace AzToolsFramework::Prefab
 
         //! Returns the size of the path to the currently focused instance.
         virtual const int GetPrefabFocusPathLength(AzFramework::EntityContextId entityContextId) const = 0;
+
+        //! Sets the current focus mode.
+        virtual void SetPrefabEditScope(AzFramework::EntityContextId entityContextId, PrefabEditScope mode) = 0;
```

Comment:
nit: fix comment and argument name (this used to be called "FocusMode" originally, but I changed its name down the line and probably forgot to update these)

Reply:
Comment/argument name updated.